config/env_spec_request.go

```diff
@@ -321,7 +321,7 @@ func (e *EnvironmentSpecRequest) getAuthenticationRequirement() (auth Authentica
 
 // IsAuthorizationRequired returns true if Authorization is required.
 func (e *EnvironmentSpecRequest) IsAuthorizationRequired() bool {
-	return !e.GetConsumerAuthorization().isEmpty()
+	return !e.GetConsumerAuthorization().isEmpty() && !e.GetConsumerAuthorization().Disabled
```

Comment:
Reversing the checks would be more efficient.

Reply:
Done. Thanks.